### PR TITLE
NAS-130767 / 25.04 / Optimize app.query calls

### PR DESCRIPTION
**Pull request by denysbutenko**
**Changes:**

- Reduce usage `app.query` endpoint
- Use `app.available_space` for retrieving available space in ix-apps dataset 

**Testing:**

See ticket for details

### CODE REVIEW

**Review by RehanY147 — CHANGES_REQUESTED** [review]
When I install a new app, for the first time, the installed-apps page doesn't update. It continues to show me empty message.

File: src/app/pages/apps/store/installed-apps-store.service.ts [hunk 1]
```diff
@@ -69,119 +64,29 @@ export class InstalledAppsStore extends ComponentStore<InstalledAppsState> imple
...
-      filter(
-        ([loading, isDockerStarted]) => {
-          return !loading && isDockerStarted !== null;
-        },
-      ),
-      tap((isDockerStarted) => {
-        if (isDockerStarted) {
-          this.subscribeToInstalledAppsUpdates();
-        }
+      debounceTime(300),
```
[the remark this answers is not in the document]
Comment (denysbutenko) [reply src/app/pages/apps/store/installed-apps-store.service.ts:72]: Coz observable emits two values in a short period that causes two websocket calls.

**Review by RehanY147 — CHANGES_REQUESTED** [review]
I don't think the debounceTime operator is the correct solution here. That just puts a patch on the problem. When using UI with a remote and slow server, the 300 millisecond could be avoided. We should instead aim to fix the condition which preceeds the `app.query` call twice.

**Comment by denysbutenko** [comment]
> I don't think the debounceTime operator is the correct solution here. That just puts a patch on the problem. When using UI with a remote and slow server, the 300 millisecond could be avoided. We should instead aim to fix the condition which preceeds the `app.query` call twice.

Well, I think it is related to this:

```
readonly isDockerStarted$ = this.select((state) => {
    return [
      DockerStatus.Initializing,
      DockerStatus.Pending,
      DockerStatus.Running,
    ].includes(state.statusData.status as DockerStatus);
  });
```

The `isDockerStarted` checks for non-started statuses and emits intermediate values.
Let's scope `isDockerStarted` to `DockerStatus.Running` only and remove `debounceTime` operator.

**Comment by denysbutenko** [comment]
Okay, I found the root cause. There was a check for a boolean when the response was an array of booleans. That was fixed.

**Review by RehanY147 — APPROVED** [review]

**Review by AlexKarpov98 — APPROVED** [review]

**Pull request merged**